### PR TITLE
Share application functionality

### PR DESCRIPTION
**Pull request by enrichp**
We'll have to change the download link at the end of the sprint.

### CODE REVIEW

File: app/src/main/java/org/pesmypetcare/mypetcare/activities/MainActivity.java [hunk 1]
```diff
@@ -630,15 +632,32 @@ private void initializeActivity() {
         setUpNavigationDrawer();
         setStartFragment();
         setFloatingButtonListener();
+        setUpShareAppListener();
         hideWindowSoftKeyboard();
     }
 
+    private void setUpShareAppListener() {
```
Comment (xavier-campos-diaz) [inline app/src/main/java/org/pesmypetcare/mypetcare/activities/MainActivity.java:642]: You should comment this method

**Review by xavier-campos-diaz — COMMENTED** [review]
You have some minor issues

**Review by AlbertPG — DISMISSED** [review]
Taking into account the previous comment, I see no other issue

**Review by AlbertPG — APPROVED** [review]
Okey I approve it

**Review by xavier-campos-diaz — APPROVED** [review]

**Pull request merged**